distribution/docs/src/main/resources/_contents/quickstart-contents.adoc

```diff
@@ -95,7 +95,7 @@ ${branding-lowercase}${at-symbol}local>
 ==== Catalog Capabilities
 
 . Download a sample valid https://codice.atlassian.net/wiki/download/attachments/1179756/geojson_valid.json?version=1&modificationDate=1368249436010&api=v2[GeoJson file here].
-. Navigate in the browser to the Search UI at http://localhost:8993/search.
+. Navigate in the browser to the Search UI at ${secure_url}/search.
```

Comment:
actually, it was a typo before. `http` doesn't use `8993`, so it wouldn't have worked as written